src/pages/Authenticate/Authenticate.tsx

```diff
@@ -61,7 +64,7 @@ const Authenticate: React.FC = () => {
         </IonItem>
         {error && (
           <div role="alert" style={{ color: "var(--ion-color-danger)" }}>
-            Something went wrong
+            {error.message}
```

Comment:
side note: we'll still want to localise this (e.g if you run GoodChat in chinese)